.github/workflows/add-to-project.yml

```diff
@@ -0,0 +1,18 @@
+name: Add new issues to project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add new issues to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          # You can target a repository in a different organization
+          # to the issue
+          project-url: https://github.com/users/mattahrens/projects/2
+          github-token: ${{ secrets.MATTAHRENS_TOKEN }}
```

Comment:
try use `secrets.PROJECT_TOKEN` for prod repo, I just added this one to cover projects ops